### PR TITLE
LITE-23560 support decorators for events, schedulables and variables

### PR DESCRIPTION
**Pull request by ffaraoneim**

### CODE REVIEW

**Review by marcserrat — APPROVED** [review]

File: connect/eaas/runner/config.py [hunk 1]
```diff
@@ -155,7 +155,7 @@ def update_dynamic_config(self, data):
                 data.event_definitions or self.dyn_config.event_definitions
             )
 
-        logger.info(f'Runner dynamic config updated {data}')
+        logger.debug(f'Runner dynamic config updated {data}')
```
Comment (Sainomori) [inline connect/eaas/runner/config.py:158]: 👍

File: connect/eaas/runner/handler.py [hunk 2]
```diff
@@ -1,85 +1,133 @@
...
+                self.get_logger(task_id),
             ),
-            default_headers=self.config.get_user_agent(),
+            default_headers=self._config.get_user_agent(),
         )
+
+    def get_events(self):
+        if 'capabilities' in self._descriptor:
+            logger.warning(
+                "The definition of extension's capabilities in extension.json is deprecated.",
```
Comment (Sainomori) [inline connect/eaas/runner/handler.py:98]: 👍

File: connect/eaas/runner/handler.py [hunk 3]
```diff
@@ -1,85 +1,133 @@
...
+
+        return {
+            event['event_type']: event
+            for event in self._extension_class.get_events()
+        }
+
+    def get_schedulables(self):
+        if 'schedulables' in self._descriptor:
+            logger.warning(
+                "The definition of extension's schedulables in extension.json is deprecated.",
```
Comment (Sainomori) [inline connect/eaas/runner/handler.py:118]: I Like deprecation messages so much! 👍

File: tests/test_worker.py [hunk 4]
```diff
@@ -248,6 +251,142 @@ def process_asset_purchase_request(self, request):
     )
 
 
+@pytest.mark.asyncio
```
Comment (Sainomori) [inline tests/test_worker.py:254]: 👍

**Review by Sainomori — APPROVED** [review]

**Review by bdjilka — APPROVED** [review]

**Pull request merged**